packages/discord.js/src/managers/GuildChannelManager.js

```diff
@@ -105,17 +105,20 @@ class GuildChannelManager extends CachedManager {
 
   /**
    * Creates a new channel in the guild.
-   * @param {string} name The name of the new channel
-   * @param {GuildChannelCreateOptions} [options={}] Options for creating the new channel
+   * @param {GuildChannelCreateOptions} options Options for creating the new channel
```

Comment:
you need to update the typedef
https://github.com/discordjs/discord.js/blob/861f0e2134662ab64a11d313130aff58b413d495/packages/discord.js/src/structures/CategoryChannel.js#L30